usr/share/drlm/conf/rear-extra/usr/share/rear/drlm-extra/restorefiles/RSYNC/default/400_restore_rsync_backup.sh

```diff
@@ -8,7 +8,7 @@
 #
 
 get_size() {
```

Comment:
I think it is the only function that would be defined within the scripts of a workflow. Shouldn't the functions be in the lib folder named WORKFLOW-functions.sh or FS-functions.sh?

Reply:
This is rear part of code so we must keep as it is. The drlm-extra code is going to be moved to rear github repo, as is part of the client side.